_posts/2021-02-19-understanding-NOPE.md

```diff
@@ -0,0 +1,177 @@
+
+This is, in general, completely incorrect.
+
+The *vast* majority of the time you are actually interacting with a **market maker**. Per [Investopedia](https://www.investopedia.com/terms/m/marketmaker.asp):
+
+> A market maker (MM) is a firm or individual who actively quotes two-sided markets in a security, providing bids and offers (known as asks) along with the market size of each. 
+
+MMs place _both_ bids and asks on the order book. Their goal is to make money from the difference between those two prices (known as the **spread**) - buy from people who want to sell _now_, sell to people who want to buy _now_, keep the change. That "keep the change" _is_ the business model of a market maker - they have no interest in holding either long or short positions, so they need to get rid of any positions (or otherwise neutralize them) ASAP. Holding a position introduces a _ton_ of risk that they don't want.
+
+In the case of trading stocks, there is typically enough volume that the amount of time they're actually holding a security - between buying it from you and flogging it to someone else - is small, so the whole thing works nicely. Impatient people get to make their trades, the spread is narrowed, they pocket some change - everybody wins. This function is so important that exchanges approve designated market makers who are _contractually obliged_ to place both bid and ask orders, in return for a commission fee.
+
+However.
+
+The "buy it and flog it to someone else quickly" approach does not work for options. The demand is just not there. Think of an options chain: a zillion dates, and a zillion strikes for each date, multiplied by two (for puts and calls). The daily volume for the most popular single option is less than the _minutely_ volume for its underlying!
+
+A market maker cannot take the other side of your trade and flog it to someone else if there's no demand for it. But they still have that pesky contractual obligation to take the trade. So they have to do something else, because being forced to take positions (that you didn't even choose!) is, um, terrible for your portfolio and not anywhere in your business model! That something else is called **hedging**, and it's the core behaviour that NOPE watches.
+
+# How do MMs avoid being stuck with the other side of my option trades?
+
+Let's say you are a market maker, and you just sold me 100 at-the-money SPY calls. You did this because you were _contractually obliged to_ - you had no choice in the matter. At the moment I buy the calls from you, this is fine, because the options were priced fairly. By fairly we mean "we both value the calls to be worth the money I paid", i.e. the cost of the premium I paid perfectly matches the risk that you're taking. You could - assuming you found a buyer - pay someone the money I just gave you to take on your obligation, and they would choose the same price.
```

Comment:
"contractually obliged to - you had no choice in the matter" a bit redundant. The "contractually obliged" is enough I think. Perhaps mention that MMs have special relationships with market participants that enforce this kind of thing

Reply:
> Perhaps mention that MMs have special relationships with market participants that enforce this kind of thing

Can you say a bit more about this? The relationship between MMs and other institutions (incl but not limited to exchanges) is definitely something outside my expertise; what I've written here is confident-sounding guesswork.

Reply:
Outside my expertise as well, and its all securities law so it is absurdly arcane and I'm sure many of the details aren't public. Now that I'm thinking it through more I think for the purposes of this the only important part is what you mentioned, the contractual obligation to have an ask and bid.